src/modules/Dashboard/features/Participants/Participants.tsx

```diff
@@ -408,8 +413,9 @@ export const Participants = () => {
   return (
     <StyledBody>
       {isLoading && <Spinner />}
-      <ParticipantsHeader>
-        <HeaderSectionLeft>
+
+      <StyledFlexWrap sx={{ gap: 1.2, mb: 2.4 }}>
```

Comment:
margins should use `theme.spacing`

Reply:
They are. 🙂 That's a [built-in shorthand](https://github.com/ChildMindInstitute/mindlogger-backend-refactor/pull/1273) of using the `sx` prop. If I had my druthers we'd be using the shorthand everywhere, but it's only used sporadically so far.